regex/src/main/scala/coop/rchain/regex/Multiplier.scala

```diff
@@ -82,6 +82,7 @@ private[regex] object Multiplier {
   * also permit a max of 0 (iff min is 0 too). This allows the multiplier
   * "zero" to exist, which actually are quite useful in their own special way.
   */
+@SuppressWarnings(Array("org.wartremover.warts.Throw"))
```

Comment:
in this particular case, because `rtegex` will be remove next week :)

in other places, sometimes there is a class (very imp3rative and ugly) and alternative would be like 5 or 6 warning. putting this all a class is an indicator "this need to be redone". i've always tried to add comment, maybe i've missed some cases?